src/CMakeLists.txt

```diff
@@ -474,6 +474,12 @@ if(FLB_BINARY)
     COMPONENT binary
     RENAME "${FLB_OUT_NAME}.conf")
 
+  install(FILES
```

Comment:
@fujimotos perhaps add it under   if (FLB_SYSTEM_WINDOWS) statement?

Reply:
> perhaps add it under if (FLB_SYSTEM_WINDOWS) statement?

@niedbalski I posted a small follow-up patch #4679 regarding this part.
Hope it helps.